Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/NotificationConstants.kt

```diff
@@ -17,6 +17,9 @@ object NotificationConstants {
     val POSITIVE_RESULT_NOTIFICATION_INITIAL_OFFSET: Duration = Duration.standardHours(2)
     val POSITIVE_RESULT_NOTIFICATION_INTERVAL: Duration = Duration.standardHours(2)
 
+    const val NEW_MESSAGE_RISK_LEVEL_SCORE_NOTIFICATION_ID = 110
+    const val NEW_MESSAGE_TEST_RESULT_NOTIFICATION_ID = 120
```

Comment:
how about defining an enum type for the notification id? this way we would have a defined set of values and it is guaranteed that caller of NotificationHelper.send / cancel passes only valid numbers

Reply:
Unfortunately, we still have the debug notifications which generate random ids. Afaik, they have been removed in 1.8. Therefore, I'd suggest postponing that refactoring to 1.8.